test/celeritas/phys/Physics.test.cc

```diff
@@ -376,7 +376,7 @@ TEST_F(PhysicsTrackViewHostTest, processes)
         ParticleProcessId const abs_ppid{1};
         EXPECT_EQ(ProcessId{0}, phys.process(scat_ppid));
         EXPECT_EQ(ProcessId{1}, phys.process(abs_ppid));
-        EXPECT_TRUE(phys.has_at_rest());
+        EXPECT_EQ(ParticleProcessId{}, phys.at_rest_process());
```

Comment:
It looks like this change is related; we want `celeriton` to have an at-rest process for testing purposes.

Reply:
Yeah, previously we were determining if a particle has an at-rest process by checking if the cross section at zero energy was positive. This wasn't really the right way to do it, e.g. some processes like photoelectric have a non-zero cross section but aren't really "at-rest" processes. I'm not sure what's the best way to identify processes that apply at rest... we might need to do something like add an `applies_at_rest()` method to the `Process` class?

Reply:
Yeah, I guess this ties into the applicability mess. If it has a nonzero cross section at zero energy, it really *should* be at rest. But we don't have a good way of creating a cross section grid that's valid down to "low-ish" energies and zero below that. Related to #907 I guess...

Reply:
I pushed a [commit](https://github.com/celeritas-project/celeritas/pull/1592/commits/cd4312e48a6b2b41451b7457a4d295d3cd7e4df6) with some ideas (flagging at-rest processes in the imported data; adding a `applies_at_rest()` method to `Process`) which I can partially or totally revert. Using imported data to determine whether the process applies at rest might be silly and we could just hardcode it, or we could try to pass that imported data directly into the `PhysicsParams` constructor somehow and avoid modifying `Process`, or do something totally different...